src/routes/search/+page.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+  import "./search-page.scss";
+  import { onMount } from "svelte";
+  import algoliasearch from "algoliasearch";
+  import instantsearch from "instantsearch.js";
+  import { configure, hits, searchBox, stats, pagination } from "instantsearch.js/es/widgets";
+  import { searchHitsTemplate } from "./searchHelpers";
+
+  const ALGOLIA_APP_ID = "9IIDYROXZ5";
+  const ALGOLIA_API_KEY = "0eac885faaa4c70dfc9dd8b6ab4ab10f";
```

Comment:
per the Algolia [docs](https://www.algolia.com/doc/guides/security/api-keys/#search-only-api-key):
> Algolia provides a search-only API key that allows you to search your data. It works on all the indices of your app and is safe to use in your production frontend code. Nevertheless, you may want to use this key to create more restricted API keys.

Reply:
Yeah, I'd love to have more confidence that this won't result in a big bill or a website with degraded functionality if someone runs wild with this. Also, even if these are public, they should be defined as environment variables instead of being hardcoded so that they can be quickly rotated without a full change process.